sway/config.c

```diff
@@ -1,4 +1,5 @@
-#define _XOPEN_SOURCE 600 // for realpath
+#define _POSIX_C_SOURCE 200809L
+#define _XOPEN_SOURCE 500 // for realpath
```

Comment:
Don't define both, they'll cause issues because `_XOPEN_SOURCE` defines implicitly `_POSIX_C_SOURCE`

Reply:
I did not know that, thanks